thumbor/threadpool.py

```diff
@@ -44,7 +43,7 @@ def __init__(self, thread_pool_size):
         return operation(*args)
 
     async def _execute_in_pool(self, operation, *args):
-        loop = asyncio.get_running_loop()
+        loop = get_running_loop()
```

Comment:
I'd suggest using `tornado.ioloop.IOLoop.current()` if possible

Reply:
Thanks! Looks for this purpose.